File: programmers/위클리 챌린지/최소직사각형.py
def solution(sizes):
    max_row = 0
    max_col = 0
    for r, c in sizes:
        if r < c: r, c = c, r
        if max_row < r: max_row = r
        if max_col < c: max_col = c
    return max_row * max_col

File: programmers/위클리 챌린지/최소직사각형.kt
class Solution {
    fun solution(sizes: Array<IntArray>): Int {
        var finalRow: Int = 0
        var finalCol: Int = 0
        sizes.map {
            var row = it[0]
            var col = it[1]
            if (it[0] < it[1]) {
                var temp = it[1]
                col = row
                row = temp
            }
            if (finalRow < row) {finalRow = row}
            if (finalCol < col) {finalCol = col}
        }
        return finalCol * finalRow
    }
}